nx/test/support/case.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Nx.Case do
```

Comment:
We already have a Nx.GradHelpers, perhaps it is worth unifying both into `Nx.Helpers`? Also, it doesn't have to be a case template. I think a module that you import as needed is enough :)